src/road_gui.cpp

```diff
@@ -1239,86 +1251,12 @@ struct BuildRoadStationWindow : public PickerWindowBase {
 		this->PickerWindowBase::Close();
```

Comment:
Might this need to be?
```suggestion
		this->PickerWindow::Close();
```